app/upgrades/v16/constants.go

```diff
@@ -5,7 +5,7 @@ package v16
 
 const (
 	// UpgradeName is the shared upgrade plan name for mainnet
-	UpgradeName = "v16.0.0"
+	UpgradeName = "v16.0.0-rc5"
```

Comment:
@MalteHerrmann this is inconsistent with the RC upgrade names below

Reply:
hmm that's true, I'll adjust this